rust/src/fvm/machine.rs

```diff
@@ -384,7 +386,12 @@ pub struct TraceMessage {
     pub codec: u64,
     pub gas_limit: u64,
     pub read_only: bool,
-    pub code_cid: Cid,
+}
+
+#[derive(Serialize_tuple, Deserialize_tuple, Debug, PartialEq, Eq, Clone)]
+pub struct TraceActor {
```

Comment:
This will require lotus changes. We introduced this change so we could determine the state of an actor _at_ the point where we invoke it (even if it never actually gets committed to the chain state, e.g., due to a revert).

The idea was to use this in the Eth JSON-RPC API.